src/main/java/tracko/model/order/Order.java

```diff
@@ -121,6 +125,20 @@ public void setDelivered() {
         this.isDelivered = true;
     }
 
+    public boolean containsItem(InventoryItem inventoryItem) {
+        return itemList.stream().anyMatch(pair -> pair.getItem().equals(inventoryItem));
+    }
```

Comment:
Would using `InventoryItem::isSameItem` be better here?

Reply:
I think equals will be better here because the `ItemQuantityPair`s should hold the `InventoryItem` reference itself, so both methods will work the same way